.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 composer.lock
-.phpunit.result.cache
\ No newline at end of file
+.phpunit.result.cache
+.idea/
```

Comment:
Custom tools files/folders, like .idea folder or vscode folder, can be ignored on global scope for developer. So maybe better approach is to not add this files/folders to gitignore, but learn each developer to setup his/her git to ignore specific folders created by his tools.